pkg/pipelineascode/cancel_pipelineruns.go

```diff
@@ -24,18 +24,21 @@ var cancelMergePatch = map[string]interface{}{
 }
 
 func (p *PacRun) cancelPipelineRuns(ctx context.Context, repo *v1alpha1.Repository) error {
-	if p.event.TriggerTarget != "pull_request" {
-		msg := fmt.Sprintf("not a pullRequest event, event: %v", p.event.TriggerTarget)
-		p.eventEmitter.EmitMessage(repo, zap.WarnLevel, "RepositoryEvent", msg)
-		return nil
-	}
+	labelSelector := getLabelSelector(map[string]string{
```

Comment:
you can just add the 		`keys.PullRequest:   strconv.Itoa(p.event.PullRequestNumber)` to the labelSelector, if `TriggetTarget == pull_request`` instead of duplicating blocks.

Reply:
Done